frontends/web/src/common/Annotation/CreateInterface.js

```diff
@@ -450,6 +451,11 @@ class CreateInterface extends React.Component {
                       });
                     }
                   )
+                  .then(() =>
```

Comment:
See comment above about whether we should have something separate to log successful model interactions